### ISSUE TITLE
Failed to compile with CRA

### ISSUE THREAD

**Issue by sergeysova** [atomixinteractions/redux-symbiote#6]
https://codesandbox.io/s/43m0nvxk34

Steps to reproduce:

1. Download sources from codesandbox
2. Install dependencies with `npm install`
3. Build `npm run build`

```
Creating an optimized production build...
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/redux-symbiote/lib/index.js:2

Read more here: http://bit.ly/2tRViJ9
```

**Comment by sergeysova** [comment]
Fixed in 1d9025f0297507bc7bd3cc7c4dd73b7975c7e5dd

**Comment by sergeysova** [comment]
Really fixed in 2.0.3

![image](https://user-images.githubusercontent.com/5620073/38332991-f8235a30-385f-11e8-83fe-a92528ad489a.png)
